alternator/stats.cc

```diff
@@ -28,19 +27,19 @@ static seastar::metrics::histogram estimated_histogram_to_metrics(const utils::e
     }
     return res;
 }
-stats::stats() : api_operations{} {
+void register_metrics(seastar::metrics::metric_groups& metrics, stats& stats) {
```

Comment:
Maybe this should be `const stats& stats` to emphasize this code only reads from stats, it never increments anything there or anything like it?

Reply:
I'll make it a single function with const reference